_search-plugins/querqy/index.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: default
+title: Querqy plugin
+has_children: false
+redirect_from:
+  - /search-plugins/querqy/
+nav_order: 10
+---
+
+# Querqy plugin
+
+Querqy is a community plugin for query rewriting that helps to solve relevance issues, making search engines more precise regarding matching and scoring.
+
+## Querqy plugin installation
+
+**Querqy is currently only compatible with OpenSearch 1.3.1**
```

Comment:
Rather than including this as bold, try this:

Querqy is currently only compatible with OpenSearch 1.3.1
{: .tip}

Reply:
converted from bold to {: .note }